src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -577,16 +578,17 @@ public boolean isStageCompatible(final String stageName) {
     checkConnection();
     InternalUtils.assertNotEmpty("stageName", stageName);
     if (!stageExist(stageName)) {
-      LOGGER.debug("stage {} doesn't exists", stageName);
+      LOGGER.info("stage {} doesn't exists", stageName);
       return false;
     }
     List<String> files = listStage(stageName, "");
     for (String name : files) {
       if (!FileNameUtils.verifyFileName(name)) {
-        LOGGER.debug("file name {} in stage {} is not valid", name, stageName);
+        LOGGER.info("file name {} in stage {} is not valid", name, stageName);
         return false;
       }
     }
+    LOGGER.info("Stage {} is valid", stageName);
```

Comment:
How about keeping it consistent with the table one? like
```suggestion
    LOGGER.info("Stage {} compatibility is {}", stageName, compatible);
```

Reply:
It required some refactor, but done